flybywiresim/a32nx
# fix(atsu): Hoppie Telex with \n can now be received

Fixes #8019

## Summary of Changes

Hoppie Telex with \n can  now be received. The Regex previously didnt capture any messages containing newline character.
Also changed the wordwrap function to keep certain characters around that are in use by virtual airlines. Also made it insert a new line if it finds a \n to try to keep formatted messages formatted.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Setup Hoppie account and fill in logon code into EFB AOC settings
2. make sure METAR and ATIS source are set to VATSIM
3. Apply power to the Aircraft
4. in MCDU insert flightnumber
5. Use a program like https://www.hoppie.nl/acars/prg/dispatch/acarsdis-2.2-install.exe 
to be able to send messages via TELEX
6. send a message like:
\*\*\*AUTOMATED UPLINK\*\*\*
GATE ASSIGNMENT FOR
FLIGHT 296
ARRIVING KFLL IS G12
GROUND POWER: YES
GROUND AIR: YES
OPS FREQ: 130.120
MESSAGE: KEEP APU
SHUTDOWN WHEN ABLE

   AUTOMATED SERVICE
PROVIDED BY VSPIRIT.IO
7. check MCDU ATSU received messages after a few seconds to see the sent message
8. check that it is all there and has the correct format
9. make sure CPDLC still works 
10. make sure Weather requests still work


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
